mlflow/R/mlflow/R/cli.R

```diff
@@ -1,29 +1,18 @@
-#' MLflow Command
```

Comment:
Remove [Roxygen-style comments](http://r-pkgs.had.co.nz/man.html#roxygen-comments) (`#'`) in favor of normal apostrophe-less R comments. Roxygen comments result in API docs being generated, but `mlflow_cli` is now private so we don't want that.